syntax = "proto2";

package api.stats;

message TeamStats {

}
